### DISCUSSION TITLE
code-server not reachable.

### DISCUSSION THREAD

**Discussion by MoinAmMorgen** [coder/code-server#6060] [Q&A]
I followed the installation guide for debian but cant connect to the website (ofc with port). If I check the status of the code-server service it shows me this:

![Bild_2023-03-08_155413564](https://user-images.githubusercontent.com/91900989/223746320-50054547-cb6f-4c55-84ed-52f918c3f7be.png)

**Comment by code-asher** [comment]
Could you try starting it with `systemctl start code-server`?

Our docs say to use `systemctl enable` but I am not actually sure 
that will start code-server immediately.

**Reply by code-asher** [reply]
Oh nevermind we say to use `systemctl enable --now` which will immediately start the service.
